andela/boondocks-bn-frontend
# #171326846 Allow user to add profile picture on profile update

#### What does this PR do?
Allows a user to update their profile picture

#### Description of Task to be completed?
- [x] update the edit profile component and actions
- [x] refactor tests

#### How should this be manually tested?
- clone this repo and run tests using `npm test`
- start the App using `npm run watch`
- navigate to your profile and you will see a placeholder image for your profile picture
- update your profile picture by uploading your new profile picture
- on successful submission, you will see your updated image rendered on the Navbar

#### Any background context you want to provide?
you will need to run the NodeJS server from [this back-end feature](https://github.com/andela/boondocks-bn-backend/pull/91) that allows this feature to work

#### What are the relevant pivotal tracker stories?
[#171326846](https://www.pivotaltracker.com/story/show/171326846)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-02-21 at 15 16 35" src="https://user-images.githubusercontent.com/17108099/75037623-49af5180-54bd-11ea-8648-ae5d43c20711.png">

<img width="1438" alt="Screen Shot 2020-02-21 at 15 17 05" src="https://user-images.githubusercontent.com/17108099/75037636-52078c80-54bd-11ea-97e7-bae328515bf2.png">

#### Questions:
N/A
